src/bci_build/package/apache_tomcat.py

```diff
@@ -121,12 +115,12 @@ def _get_sac_supported_until(
         entrypoint_user="tomcat",
         logo_url="https://tomcat.apache.org/res/images/tomcat.png",
     )
-    for tomcat_major, os_version, jre_version in (
-        (10, OsVersion.TUMBLEWEED, 22),
-        (10, OsVersion.TUMBLEWEED, 21),
-        (10, OsVersion.TUMBLEWEED, 17),
+    for tomcat_ver, os_version, jre_version in (
```

Comment:
if you are hardcoding versions already, why not include `tomcat_ver_major` and avoid any math conversions?

Reply:
deriving it from one value is more DRY than duplicating it in the tuple, imho.